python/lib/dcoscli/dcoscli/marathon/main.py

```diff
@@ -856,6 +867,34 @@ def version_list(self, app_id, max_count):
         emitter.publish(versions)
         return 0
 
+    def delay_reset(self, app_id, pod_id):
```

Comment:
only 1 argument is ever possible... this will just confuse future developers IMO